src/Entity/PackageReadme.php

```diff
@@ -0,0 +1,46 @@
+<?php declare(strict_types=1);
+
+/*
+ * This file is part of Packagist.
+ *
+ * (c) Jordi Boggiano <j.boggiano@seld.be>
+ *     Nils Adermann <naderman@naderman.de>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace App\Entity;
+
+use Doctrine\ORM\Mapping as ORM;
+
+/**
+ * @author Jordi Boggiano <j.boggiano@seld.be>
+ */
+#[ORM\Entity]
+#[ORM\Table(name: 'package_readme')]
+class PackageReadme
+{
+    public function __construct(
+        #[ORM\Id]
+        #[ORM\Column(type: 'integer', name: 'package_id')]
+        public int $packageId,
+
+        #[ORM\Column(type: 'text', nullable: true)]
```

Comment:
why is it nullable ? The Package entity expects to have no PackageReadme at all when the contents is null.